src/gafaelfawr/schema/token.py

```diff
@@ -17,7 +17,7 @@ class Token(Base):
     username = Column(String(64), nullable=False)
     token_type = Column(Enum(TokenType), nullable=False)
     token_name = Column(String(64))
-    scopes = Column(String(256))
+    scopes = Column(String(256), nullable=False)
```

Comment:
Yeah, let me double the field length just to give us some breathing room. (Unfortunately, while I'd like to use `Text`, it appears to mess up the indexing, and this has to be an indexed column in order to find internal tokens with an appropriate scope.) 256 will give us about 19 scopes, which is probably more than enough, but it doesn't hurt to make the field a bit longer.

Reply:
Oh, wait, no, I'm wrong about the index because of how internal tokens are searched for. So maybe we could use TEXT, although for now I'll leave it at 512 characters.